<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QMainWindow" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QTextBrowser" name="incomingField"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="messageField"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="sendButton">
      <property name="text">
       <string>-&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>280</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="ipPortAct"/>
    <addaction name="nameAct"/>
    <addaction name="statusAct"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="appAct"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="connectAct"/>
    <addaction name="disconnectAct"/>
    <addaction name="separator"/>
    <addaction name="saveHistoryAct"/>
    <addaction name="separator"/>
    <addaction name="quitAct"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="connectAct">
   <property name="text">
    <string>Connect to server</string>
   </property>
   <property name="toolTip">
    <string>Connect User to server by ip and port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="disconnectAct">
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect User from host</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="saveHistoryAct">
   <property name="text">
    <string>Save history to XML...</string>
   </property>
   <property name="toolTip">
    <string>Save messages history to file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="quitAct">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit application</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="ipPortAct">
   <property name="text">
    <string>Edit IP/Port...</string>
   </property>
   <property name="toolTip">
    <string>Change IP and Port for connect to host</string>
   </property>
  </action>
  <action name="nameAct">
   <property name="text">
    <string>Edit name...</string>
   </property>
   <property name="toolTip">
    <string>Edit User name</string>
   </property>
  </action>
  <action name="statusAct">
   <property name="text">
    <string>Edit status...</string>
   </property>
   <property name="toolTip">
    <string>Edit User status</string>
   </property>
  </action>
  <action name="appAct">
   <property name="text">
    <string>About app...</string>
   </property>
   <property name="toolTip">
    <string>Info about application</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
